### PR TITLE
Traces support

### PR DESCRIPTION
**Pull request by aangelisc**
This PR adds the option to format data appropriately for the trace view. It adds the required parser for the `serviceTags` and `tags` fields. It should be noted that the user will need to ensure the data returned from ADX will need to be of the same schema that the trace visualisation expects.

TODO:

- [x] - Docs updates

Fixes grafana/grafana#67454

### CODE REVIEW

**Review by asimpson — APPROVED** [review]
Looks good to me! Nice work :trophy:

**Comment by joeyj** [comment]
@aangelisc how's this PR coming?

**Comment by aangelisc** [comment]
Hi @joeyj, I've just pushed some amendments and documentation. Once approved this will be merged and released 😊

File: README.md [hunk 1]
```diff
@@ -216,6 +216,15 @@ T | make-series AvgHatInventory=avg(HatInventory) default=double(null) on Timest
   | extend series_decompose_forecast(AvgHatInventory, 30) | project-away *residual, *baseline, *seasonal
 ```
 
+### Trace Visualization Support
+
+The trace format option can be used to display appropriately formatted data using the built in trace visualization. In order to use this visualization, data must be presented following the schema that is defined [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure). The schema contains the `logs`, `serviceTags`, and `tags` fields which are expected to be JSON objects. These fields will be converted to the expected data structure provided the schema in ADX matches the below:
```
Comment (bossinc) [inline README.md:221]:
```suggestion
The trace format option can be used to display appropriately formatted data using the built in trace visualization. To use this visualization, data must be presented following the schema defined [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure). The schema contains the `logs`, `serviceTags`, and `tags` fields which are expected to be JSON objects. These fields will be converted to the expected data structure provided the schema in ADX matches one of these formats:
```

File: README.md [hunk 2]
```diff
@@ -295,11 +304,9 @@ MyLogs
...
+The query builder provides an easy to use interface to query Azure Data Explorer. As of v4.1.0 columns of type `dynamic` are also appropriately supported within the query builder. Dynamically typed columns can now be queried using the `Where`, `Aggregate`, and `Group By` operations. When choosing one of these operations the options will be appropriately populated based on the values within the dynamic column. This encompasses arrays, JSON objects, and nested objects within arrays. There are, however, some limitations; as only the first 50,000 rows are queried for data, only properties contained within the first 50,000 rows will be listed as options in the builder selectors. Also, due to the fact that these queries make use of `mv-expand` they may become resource intensive.
```
Comment (bossinc) [inline README.md:307]:
```suggestion
The query builder provides an easy to use interface to query Azure Data Explorer. As of v4.1.0, columns of type `dynamic` are also appropriately supported within the query builder. Dynamically typed columns can now be queried using the `Where`, `Aggregate`, and `Group By` operations. When choosing one of these operations, the options will be populated based on the values within the dynamic column. This encompasses arrays, JSON objects, and nested objects within arrays. A limitation is only the first 50,000 rows are queried for data, so only properties contained within the first 50,000 rows will be listed as options in the builder selectors. Also, due to the fact that these queries make use of `mv-expand`, they may become resource intensive.
```

**Review by bossinc — COMMENTED** [review]
Feel free to ignore suggestions. I never know how picky/annoying to get with docs stuff. :smile:

**Review by bossinc — APPROVED** [review]
BEAUTIFUL

**Pull request merged**